### ISSUE TITLE
Body copy Yellow underline style moved up to behave like a "highlight" on paper

### ISSUE THREAD

**Issue by patfinney** [theseangoodwin/theseangoodwin.github.io#19]
possible to bump the yellow stroke up to just barely intersect the text? I'm thinking minimal overlap, less than 1/3 of the text height.

**Comment by theseangoodwin** [comment]
Yes, I'm all for this. I think this is something you initially pushed back against when we were making that line. :)

**Comment by theseangoodwin** [comment]
<img width="796" alt="screen shot 2017-07-15 at 12 53 42 pm" src="https://user-images.githubusercontent.com/25349814/28241143-ae84ac44-695c-11e7-9169-f809181db191.png">

**Comment by theseangoodwin** [comment]
I have a little bit of a `skewY()` on this, `1.5deg`
